Makefile

```
@@ -3,8 +3,9 @@ sync_translations sync_translations_github_workflow
 
 
 # Default languages for the sync_translations.py file
-# Eventually, this should be all the languages Open edX supports. We're starting with a subset for now.
-export TX_LANGUAGES := ar,de,fr_CA
+# This includes languages which are translatie
+# This list should be all the languages Open edX supports until the migration is complete.
+export TX_LANGUAGES := ar,da,de_DE,el,es-149,es_ES,fr_CA,hi,id,it_IT,pt_BR,pt_PT,ru,th,tr_TR,uk,zh_CN
```

Hebrew is in [the table](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3157524644/Translation+Working+Group#The-following-is-a-table-of-the-latest-list-of-languages-supported--by-the-Translation-Working-Group) but without a code
```suggestion
export TX_LANGUAGES := ar,da,de_DE,el,es-149,es_ES,fr_CA,hi,he,id,it_IT,pt_BR,pt_PT,ru,th,tr_TR,uk,zh_CN
```

---

I've fixed the table